### PR TITLE
added functionality to edit status card

### PR DESCRIPTION
**Pull request by upatricck**
## DONE
* Functionality to edit status_card
* Enhcancing the ui

### CODE REVIEW

**Comment by jlark** [comment]
screenshot please

**Comment by upatricck** [comment]
screenshot from 2016-12-19 15 11 15

**Comment by upatricck** [comment]
screenshot from 2016-12-19 16 30 07

File: seshdash/views.py [hunk 1]
```diff
@@ -116,6 +116,11 @@ def index(request,site_id=0):
         measurements[measurement] = measurement_verbose_name
     context_dict['measurements']= measurements
 
+    # status card form
```
Comment (upatricck) [inline seshdash/views.py]: We are using the choices field in the Status Card Model

**Comment by jlark** [comment]
LGTM

**Pull request merged**